RecoLocalTracker/SubCollectionProducers/src/SeedClusterRemover.cc

```diff
@@ -111,7 +111,9 @@ void SeedClusterRemover::readPSet(
 }
 
 SeedClusterRemover::SeedClusterRemover(const ParameterSet &iConfig)
-    : doStrip_(iConfig.existsAs<bool>("doStrip") ? iConfig.getParameter<bool>("doStrip") : true),
+    : tTrackerGeom_(
+          esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>(edm::ESInputTag("", ""))),  //is it correct to use "" ?
```

Comment:
the original comment was made 6 years ago.
https://github.com/cms-sw/cmssw/commit/65ded94802f92148331ede292e8a496fa105f226#diff-34ddaf02250754d45c345cbe666b5e9ef429c81f61cc386f5a2761a1fc7b9829R385

is it reasonable to expect that the comment is not necessary anymore? (there is no default argument value to escape passing something)

Reply:
I suppose it can. If I remove the comment, I think I can remove entirely the `ESInputTag` as I think that `("","")` it's the default.

Reply:
OK then, even simpler/shorter